### PR TITLE
fixed and updated strings.xml with Dutch translation

### PR DESCRIPTION
**Pull request by Pandemic-XDA**
all good SEpolicy etc is inside and all is good now

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -20,132 +20,147 @@
 
 <resources>
     <string name="app_name" translatable="false">LSPosed</string>
+
+    <!-- MainActivity -->
+    <string name="info">Info</string>
+    <string name="Activated">Geactiveerd</string>
```
Comment (tehcneko) [inline app/src/main/res/values-nl/strings.xml:26]: this was removed

File: app/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -20,132 +20,147 @@
...
+    <string name="group_network">Netwerk</string>
+    <string name="dns_over_http">DNS via HTTPS</string>
+    <string name="dns_over_http_summary">Tijdelijke oplossing voor DNS-vergiftiging in sommige landen</string>
+    <string name="theme_color">Thema kleur</string>
+    <string name="theme_color_summary">Kleur die het meest voorkomt in de app</string>
+	  <string name="dark_theme">Donkere thema</string>
+	  <string name="dark_theme_follow_system">Volg systeem</string>
+	  <string name="dark_theme_off">Altijd uit</string>
+	  <string name="dark_theme_on">Altijd aan</string>
+
```
Comment (tehcneko) [inline app/src/main/res/values-nl/strings.xml:162]: formatting

**Review by tehcneko — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-nl/strings.xml [hunk 3]
```diff
@@ -20,132 +20,147 @@
 
 <resources>
     <string name="app_name" translatable="false">LSPosed</string>
+
+    <!-- MainActivity -->
+    <string name="info">Info</string>
```
[the remark this answers is not in the document]
Comment (Pandemic-XDA) [reply app/src/main/res/values-nl/strings.xml:25]: oke then i edit it

File: app/src/main/res/values-nl/strings.xml (hunk 2 above)
Comment (tehcneko) [reply app/src/main/res/values-nl/strings.xml:162]: image

Comment (Pandemic-XDA) [reply app/src/main/res/values-nl/strings.xml:162]: thats fixed also

**Comment by Pandemic-XDA** [comment]
All is corrected now ;)

**Review by Pandemic-XDA — COMMENTED** [review]
fixed

**Pull request merged**